wasm-corpus/run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# run with `rebuild` to build the image
+# run with no args to compile everything
+
+if [[ "$1" = "rebuild" ]]; then
+  docker build . --platform linux/amd64 -t chicory/wasm-corpus
+else
+  # Optionally takes the args `lang` (ex: wat, rust) and `file` (ex: br.wat)
+  # both default to all
+  docker run --platform linux/amd64 -v ./src/test/resources:/usr/code --rm chicory/wasm-corpus $1 $2
```

Comment:
I have used this mechanism in the past and, on some systems, the file permissions are going to be messed up, still, we can fix this later on.